src/client/js/CoreEditor.js

```diff
@@ -376,14 +414,27 @@ class LinkUrlControls extends React.Component {
     }
     render() {
         return (
-            <div style={styles.urlInputContainer}>
+            <div style={styles.linkTitleInputContainer}>
+                <label>Title: </label>
+                <input
+                  onChange={this.onTitleChange}
+                  ref="linkTitle"
+                  style={styles.linkTitleInput}
+                  type="text"
+                  value={this.props.titleValue}
+                  onKeyDown={this.onTitleInputKeyDown}
+                />
+                <label>Url: </label>
+                <input
+                  ref="url"
+                  style={styles.linkURLInput}
+                  type="text"
+                />
+                <label>Description: </label>
```

Comment:
@muddaserahmed Use bootstrap forms forms for dialog formatting
http://www.w3schools.com/bootstrap/bootstrap_forms_inputs.asp